app/routes/member/invites/$invitationId/index.tsx

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import { LoaderArgs, redirect } from "@remix-run/node";
+import { requireMemberId, getMemberId } from "~/memberSession.server";
+import { getInvitation } from "~/models/invitation.server";
+import { useLoaderData, Link, Form, useTransition } from "@remix-run/react";
+import parseISO from "date-fns/parseISO";
+import format from "date-fns/format";
+import formatDistance from "date-fns/formatDistance";
+import { FaQuestion, FaCheck, FaExclamation } from "react-icons/fa";
+import { ImCross } from "react-icons/im";
+import { useState } from "react";
+
+export async function loader({ request, params }: LoaderArgs) {
+  await requireMemberId(request);
+
+  const invitationId = params["invitationId"];
+
+  if (!invitationId) {
+    throw redirect("/");
```

Comment:
Probably best to redirect them to the member dashboard instead of the homepage, yeah?